job_spec/INSAR_ISCE.yml

```
@@ -84,6 +84,12 @@ INSAR_ISCE:
           - GMAO
           - HRES
           - HRRR
+    min_frame_coverage:
+      api_schema:
+        description: Minimum amount of common area the reference and secondary granules overlap. Default is 0.01.
```

Something like this? Also, not sure we need to include the default value in the description.
```suggestion
        description: Minimum amount of the frame that must be covered by the overlap of the reference and secondary granules.
```